language-support/ts/codegen/src/TsCodeGenMain.hs

```diff
@@ -340,6 +343,7 @@ genType curModName = go
         TCon _ -> error "IMPOSSIBLE: lonely type constructor"
         TSynApp{} -> error "IMPOSSIBLE: type synonym not serializable"
         t@TApp{} -> error $ "IMPOSSIBLE: type application not serializable - " <> DA.Pretty.renderPretty t
+          -- The above case also handles 'TNumeric a' where 'a' is not a type level nat.
```

Comment:
I don't think this comment really helps. Let's just drop it.

Reply:
✔️